server/bin/pbench-remote-sync-package-tarballs

```diff
@@ -7,8 +7,8 @@
 opts=$SHELLOPTS
 case $opts in
     *xtrace*)
-        dir=$(dirname $(which $0))
-        PROG=$(basename $(which $0))
+        dir=$(dirname $(command -v $0))
+        PROG=$(basename $(command -v $0))
```

Comment:
I don't think this case is tested by *any* unit tests. Can you make sure that it works properly? Manually if necessary, but ideally by adding a unit test.

Reply:
Perhaps we can just consider adding an assertion that `${dir}` and `${PROG}` are properly defined by line 18.  If we do that, then we'll save ourselves a lot of headaches, and we won't necessarily need a unit test for that.  What do you think, @ndokos?

Reply:
That'd be great.

Reply:
Do you mean to check whether the var are empty?

Reply:
Yes, so something like the following immediately following line 26:

```
if [[ -z "${dir}" || -z "${PROG}" ]]; then
    echo "INTERNAL ERROR - bad environment encountered" >&2
    exit 1
fi
```

Reply:
And if you could also fix the `echo` statement on line 23 to send its output to stderr using, "`echo "Usage: ..." >&2`", that would be great.